my-react-project/src/App.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+
+function App() {
+  const [count, setCount] = useState(0)
+
+  return (
```

Comment:
Heheh, this isn't the music project right? I think the files are a bit jumbled with the tuesday-exercise codes 😅